manifests/vhost.pp

```diff
@@ -338,23 +338,24 @@
   }
 
   if $ip {
+    $_ip = any2array($ip)
     if $port {
-      $listen_addr_port = "${ip}:${port}"
-      $nvh_addr_port = "${ip}:${port}"
+      $listen_addr_port = parseyaml(inline_template('<% res="" %><% @_ip.each { |a_ip| res = res + "#{a_ip}:#{@port}," } %><%= res.split(",").to_yaml -%>'))
```

Comment:
we'll have a real hard time porting this module to 4.x because it's pretty much one of the most used ones out there… i don't really have a strategy there…